CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix broken fragments on CodeSandbox @lucivpav ([#1655](https://github.com/stardust-ui/react/pull/1655))
 - Improve a table with component props in docs @layershifter ([#1634](https://github.com/stardust-ui/react/pull/1634))
 - Improve rendering performance for docs examples @Bugaa92 ([#1654](https://github.com/stardust-ui/react/pull/1654))
+- Add FAQ entry for Flex, Grid, Layout, Box, Segment @lucivpav ([#1646](https://github.com/stardust-ui/react/pull/1646))
```

Comment:
nit: please, embrace component names in back-ticks, for the sake of formatting consistency in the changelog